src/data_channel/data_channel_test.rs

```diff
@@ -603,3 +604,57 @@ async fn test_stats() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_poll_data_channel() -> Result<()> {
+    let mut sbuf = vec![0u8; 1000];
+    let mut rbuf = vec![0u8; 1500];
+
+    let (br, ca, cb) = Bridge::new(0, None, None);
+
+    let (a0, a1) = create_new_association_pair(&br, Arc::new(ca), Arc::new(cb)).await?;
+
+    let cfg = Config {
+        channel_type: ChannelType::Reliable,
+        reliability_parameter: 123,
+        label: "data".to_string(),
+        ..Default::default()
+    };
+
+    let dc0 = Arc::new(DataChannel::dial(&a0, 100, cfg.clone()).await?);
+    bridge_process_at_least_one(&br).await;
+
+    let dc1 = Arc::new(DataChannel::accept(&a1, Config::default()).await?);
+    bridge_process_at_least_one(&br).await;
+
+    let mut poll_dc0 = PollDataChannel::new(dc0.clone());
+    let mut poll_dc1 = PollDataChannel::new(dc1.clone());
+
+    sbuf[0..4].copy_from_slice(&1u32.to_be_bytes());
```

Comment:
Perhaps using a more "tricky" value than `1` here would be more robust. e.g. a random value with bits set in all octets such as `961666284`